apache/zeppelin
# [Zeppelin-1496] Apply Zeppelin-Web Good Practice Guide #1 to the code

### What is this PR for?
This is some refactoring of zeppelin-web to apply the IIFE as stated in the Good Practice Guide #1 (https://zeppelin.apache.org/contribution/zeppelinweb/goodPracticeGuide01.html)

### What type of PR is it?
Refactoring

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-1496

### How should this be tested?
Run The webapp, and go around the page, things should work like usual.
But overall, just need to check that there is no type or parameter mismatch during the refactoring process (controller to controller, params in $inject etc...)

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No
